src/api/admin.rs

```diff
@@ -22,4 +27,19 @@ pub fn admin_service() -> impl HttpServiceFactory {
     web::scope("/admin-api")
         .wrap(HttpAuthentication::bearer(validator))
         .service(web::resource("/v2/updates/").route(web::post().to(update_bcd)))
+        .service(web::resource("/ai-history/").route(web::post().to(delete_old_ai_history)))
+}
+
+pub async fn delete_old_ai_history(
+    pool: Data<Pool>,
+    arbiter: Data<ArbiterHandle>,
+) -> Result<HttpResponse, ApiError> {
+    if !arbiter.spawn(async move {
+        if let Err(e) = do_delete_old_ai_history(pool).await {
+            error!("{}", e);
+        }
```

Comment:
Just wondering out loud if this would be "more" idiomatic:

```suggestion
        do_delete_old_ai_history(pool)
            .await
            .map_err(|e| error!("{}", e))
            .ok();
```

Reply:
Yes, it looks more rustful.